tcf_website/management/commands/upload_csv_review_drive.py

```diff
@@ -0,0 +1,174 @@
+from django.core.management.base import BaseCommand, CommandError
+
+from tcf_website.models import *
+from datetime import datetime
+import csv
+import os
+
+DATA_DIR = "tcf_website/management/commands/review_drive_responses/"
+
+class Command(BaseCommand):
+
+    # Run this coomand using `sudo docker exec -it tcf_django python3 manage.py upload_csv_review_drive [filename] [season] [year]`
+    # Optional flag --verbose can be set to show output of the script
+    # Must be run in a separate terminal after already running docker-compose up
+    # May have to restart the docker container for database changes to be visible on the site
+    # Visually check that CS department has been moved to E School and that
+    # other schools have been split up.
+
+    help = "Uploads CSV from /review_drive_responses/ in the format of Time, Email, Mnemonic, Number, Name, Review, " \
+           "Rating, Enjoyable, Recommendation, Difficulty, Reading Hours, Writing Hours, Group work Hours, Misc Hours"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "filename",
+            help=(
+                'Filepath of csv to upload in /review_drive_responses/'
+                "Must be in correct format."
+            ),
+            type=str,
+        )
+        parser.add_argument(
+            "season",
+            help=(
+                'What season the reviews took place i.e. Fall, January, Spring, Summer'
+            ),
+            type=str,
+        )
+        parser.add_argument(
+            "year",
+            help=(
+                'Year of reviews i.e. 2024'
+            ),
+            type=int,
+        )
+
+        # Named (optional) arguments
+        parser.add_argument(
+            "--verbose",
+            action="store_true",
+            help="Verbose output",
+        )
+
+    def handle(self, *args, **options):
+        verbose = options["verbose"]
+        filename = options["filename"]
+        season = options["season"]
+        year = options["year"]
+
+        # Get semester object from args
+        semester_query = Semester.objects.filter(season=season.upper(), year=year)
+        if semester_query.count() <= 0:
+            print('ERROR: Semester not found')
+            return
+        semester = semester_query.first()
+
+        # Needs to be created ahead of time in the db
```

Comment:
What are we doing to ensure this user will always be created?

Reply:
The command does not run if the user doesn't exist, lines 67-70 covers this.

Reply:
Yes - my point is what if, somehow, the user is *not* created and the command is run. Do we want to create the dummy user instead of just failing?

Reply:
I was thinking of doing that, but we don't want the user password to be in the code - it's open source. We can consider adding it to the env if it's important enough, but after it gets created in the database once, there's no reason to have code to auto-add it. If you think it's important enough, I'll connect it to the .env to auto-create a user.

Reply:
If you don't think it's necessary, go ahead and resolve this.

Let's say we have something irrecoverable happen to our DB - I think all of our commands and processes should be functioning as expected and not require us to reimplement changes.

For example, when adding trigram to the DB, I made sure to have a migration that enables the extension (even though, realistically, we should never need to enable the extension again).

Reply:
Added an auto-creation from the .env.